performance/benchmarks/bm_telco_fractions.py

```diff
@@ -0,0 +1,100 @@
+#-*- coding: UTF-8 -*-
+
+""" Telco Benchmark for measuring the performance of Fraction calculations
+
+http://www2.hursley.ibm.com/decimal/telco.html
+http://www2.hursley.ibm.com/decimal/telcoSpec.html
+
+A call type indicator, c, is set from the bottom (least significant) bit of the duration (hence c is 0 or 1).
+A r, r, is determined from the call type. Those calls with c=0 have a low r: 0.0013; the remainder (‘distance calls’) have a ‘premium’ r: 0.00894. (The rates are, very roughly, in Euros or dollarates per second.)
+A price, p, for the call is then calculated (p=r*n).
+A basic tax, b, is calculated: b=p*0.0675 (6.75%), and the total basic tax variable is then incremented (sumB=sumB+b).
+For distance calls: a distance tax, d, is calculated: d=p*0.0341 (3.41%), and then the total distance tax variable is incremented (sumD=sumD+d).
+The total price, t, is calculated (t=p+b, and, if a distance call, t=t+d).
+The total prices variable is incremented (sumT=sumT+t).
+The total price, t, is converted to a string, s.
+
+"""
+
+from struct import unpack
+import os.path
+
+import perf
+
+
+def rel_path(*path):
+    return os.path.join(os.path.dirname(__file__), *path)
+
+filename = rel_path("data", "telco-bench.b")
```

Comment:
I suggest:

DATA_FILE = os.path.join(os.path.dirname(__file__), "data", "telco-bench.b")

Oh, I see that this code is also used in the current bm_telco.py :-) So maybe keep it. I might change it later ;-)